src/esm_runscripts/filedicts.py

```diff
@@ -63,10 +65,40 @@ def cp(self) -> None:
     def ln(self) -> None:
         pass
 
-    def mv(self) -> None:
-        pass
-        
-        
+    def mv(self, source, target) -> None:
+        """
+        Moves (renames) the SimulationFile from it's location in `source` to
+        it's location in `target`.
+
+        Source and target should be on of work, pool, exp_tree, or run_tree.
+        You need to specify name_in_`source` and name_in_`target` in the
+        object's attrs_dict.
+        """
+        if source not in self.locations:
+            raise ValueError(
+                "source is incorrectly defined, and needs to be in {self.locations}"
+            )
+        # NOTE(PG): The next few lines can probably be a internal function, we
+        # will use it in copy, move, and link:
+
+        # Figure out names in source and target:
+        sname = self.get(f"name_in_{source}")
+        tname = self.get(f"name_in_{target}")
+        # Relative path in source and target
+        srpath = self.get(f"path_in_{source}")
+        trpath = self.get(f"path_in_{target}")
+        # Build target and source paths:
+        spath = self.locations[source].joinpath(srpath, sname)
```

Comment:
we can use little bit more verbosity like `source_path`, `target_path` and let vim do the autocomplete with `ctrl-n`